lib/average_end_of_day_loan_balance/main.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import asyncio
+import os
+
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd  # type: ignore
+
+from pngme.api import AsyncClient
+
+
+async def get_average_end_of_day_balance(
+    api_client: AsyncClient,
+    user_uuid: str,
+    utc_starttime: datetime,
+    utc_endtime: datetime,
+) -> Optional[float]:
+    """Calculates the average end-of-day total balance for a user across all
+       of their accounts for a given time window.
```

Comment:
specify it is for their 'loan accounts' here?

Reply:
Done. Also added the one for depository.